api/mapping/services_nlp.py

```diff
@@ -67,12 +353,12 @@ def nlp_single_string(pk, dict_string):
 
     return True
 
+
 def get_json_from_nlpmodel(json):
-    
     """
     A small function to process the JSON string saved in NLPModel
     """
-    
+
     # Define which codes we want to keep. Add more here as required.
     codes = []
     keep = ["ICD9", "ICD10", "RXNORM", "SNOMEDCT_US"]
```

Comment:
I think this list is defined multiple times. Can you extract it to a common place?

Reply:
@py5gol I've been meaning to speak to you about this. I think I mentioned it previously but I was wondering if we ought to keep our list of vocabs in a mini-model so they are available across the whole site (if needed). Might be a bit overkill, so let me know what you think.

Reply:
As an aside, now that the old functions at the bottom of services_nlp.py have been removed, this list does only apear once in the entire NLP code. A model may therefore be overkill. Let me know what you think @py5gol